app/scripts/components/table/list-view.js

```diff
@@ -189,7 +189,7 @@ class List extends React.Component {
       bulkActionError
     } = this.state;
     const primaryIdx = 0;
-    const hasActions = Array.isArray(bulkActions) && bulkActions.length;
+    const hasActions = Array.isArray(bulkActions) && bulkActions.length > 0;
```

Comment:
Honest question: What is the difference between testing `bulkActions.length` and `bulkActions.length > 0?

Reply:
It's a matter of a specific "truthy" value. Without the addition of `> 0`, the value assigned to `hasActions` is the actual length of `bulkActions`, not a boolean value. The problem with that is that when you are then forced to coerce it to a boolean value when you want an actual boolean value, which is what was being done on the old line 234: `!!hasActions`. Now that `> 0` is added, `hasActions` is a true boolean value and I can change `!!hasActions` to `hasActions` (which I did), which is arguably more intuitive and readable.

Reply:
Good catch and explanation 👍